### PR TITLE
TASK-19: SonarCloud

### PR DESCRIPTION
**Pull request by cindyslittleplanet**
This PR implements TASK-19 (#60)

Instead of SonarQube, we switched over to SonarCloud and it runs on Travis.

Link to SonarCloud: https://sonarcloud.io/dashboard?id=RGPosadas_WayFinder

Note: No more Wiki is needed since there is no installation for SonarQube or Sonar Scanner.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,10 +9,16 @@ before_install:
 before_script:
   - curl https://pre-commit.com/install-local.py | python -
   - pre-commit install
+addons:
+  - sonarcloud:
+    organization: "rgposadas"
+    token:
+      secure: $SONAR_TOKEN
```
Comment (JimLau49) [inline .travis.yml:16]: Is this the correct name of our token?

File: package.json [hunk 2]
```diff
@@ -1,16 +1,6 @@
 {
```
Comment (cristian-aldea) [inline package.json]: Please run `npx sort-package-json` to sort this package.json. Stuff seems to have moved around

**Review by cristian-aldea — CHANGES_REQUESTED** [review]
Everything looks good! I just had one small comment which will be quick to fix.

Also, is there a way I can run something to check if it works? or I just need to consult the given dashboard?

File: .travis.yml (hunk 1 above)
Comment (cindyslittleplanet) [reply .travis.yml:16]: Yes, cause the token is a private environment variable saved in our Travis build.

File: package.json (hunk 2 above)
Comment (cindyslittleplanet) [reply package.json]: The package.json has been cleaned up. 
Sonar-scanner will only run and update the board if a commit is pushed or a PR is created, so yes, you just need to consult the SonarCloud dashboard.

**Review by JimLau49 — DISMISSED** [review]
LGTM! 👍

**Review by RGPosadas — CHANGES_REQUESTED** [review]
This is super cool to have!

However, Travis is inundated with `WARNING` and `INFO` level logs. 
Ex (and this isn't even all of them):
image

Is it possible to suppress some of these, and keep important `INFO` ones?
Like this one:
image

**Review by r-saba — COMMENTED** [review]
SonarQube dashboard says 75% line coverage but when I run the tests locally I see 83% line coverage is this normal?

**Review by cristian-aldea — DISMISSED** [review]
The changes look good! Having a nice dashboard like this will really help us for the last sprint.

**Review by cristian-aldea — DISMISSED** [review]
The changes look good! My review was invalidated by the new pushes but it still stands!

**Review by cristian-aldea — APPROVED** [review]
LGTM!

**Review by RGPosadas — APPROVED** [review]
My concerns have been addressed. LGTM!

**Pull request merged**